File: C#_B-493/Multiarr.cs
using System;

namespace Array
{
	class Multiarr
	{
		public static void Main()
		{
			Console.Write("Enter size of row : ");
			int row = Convert.ToInt32(Console.ReadLine());
			Console.Write("Enter size of column : ");
			int col = Convert.ToInt32(Console.ReadLine());
			int [,] arr = new int[row,col];
			
			//Console.WriteLine(arr.GetLength(1));
			Console.WriteLine("\n\nEnter value of array : ");
			
			for(int i=0;i<arr.GetLength(0);i++)
			{
				for(int j=0;j<arr.GetLength(1);j++)
				{
					Console.Write("arr[{0},{1}] : " ,i,j);
					arr[i,j] = Convert.ToInt32(Console.ReadLine());		
				}
			}
			
			Console.WriteLine("\n\nPrint value of array : ");
			
			for(int i=0;i<arr.GetLength(0);i++)
			{
				for(int j=0;j<arr.GetLength(1);j++)
				{
					//Console.WriteLine("arr[{0},{1}] : {2}" ,i,j,arr[i,j]);
					Console.Write(arr[i,j] + "\t" );
				}
				Console.Write("\n");
			}
			Console.ReadLine();
			
		}
	}
}

File: java_B-493/Multiarr.java
import java.util.Scanner; 

class Multiarr
{
	public static void main(String[] args)
	{
		
		Scanner object = new Scanner(System.in);
		
		System.out.print("\nEnter size of row : ");
		int row = object.nextInt();	

		System.out.print("\nEnter size of column : ");
		int col = object.nextInt();	
		
		int [][] arr = new int[row][col];
		
		//int [][] arr = new int[2][2];	
		
		/*arr[0][0] = 1;
		arr[0][1] = 2;
		arr[1][0] = 3;
		arr[1][1] = 4;*/
		
		System.out.println("\nEnter value of array : ");
			
		for(int i=0;i<arr.length;i++)
		{
			for(int j=0;j<arr[i].length;j++)
			{
				System.out.print("arr [" + i + "][" + j + "]: ");
				arr[i][j] = object.nextInt();
			}
		}
		
		System.out.println("\n\nPrint value of array : ");
		
		for(int i=0;i<arr.length;i++)
		{
			for(int j=0;j<arr[i].length;j++)
			{
				System.out.println("arr [" + i + "][" + j + "] : " + arr[i][j]);
			}
		}
		
	}
}